### ISSUE TITLE
Music not playing but skipping through playlist

### ISSUE THREAD

**Issue by psiebe** [athombv/homey#1554]
**What is your Homey version (Settings → System)?**

1.2.2

**What did you try to achieve?**
I want to play music from a flow, this occasionally works, but often does not result in music. 

*Please include detailed steps how we can reproduce the issue*
My setup includes 2 Sonos Play 1 (app version 2.3.0), and a premium spotify account (1.0.6).

I have defined a flow where I hit a switch and homey sets an active speaker and starts playing a sonos playlist. Sometimes this works flawless. sometimes there is no music.

If there is no music and I open the Homey app and goto music, I can select the playlist (does not matter whether it's a sonos playlist or a spotify for sonos playlist)  I can see the playlist being refreshed/populated in the main window. When I hit play, or select a number to play it shows as playing (no sound though) for about 1-2 seconds, then skips to the next song and on and on until the playlist is done, then starting again, never producing any sound, never spending more then 2 seconds on a song.

Tried this with multiple playlists, long and short, and it sometimes works great after a new restart, but the issues seems to come back

**What was the result?**

...see above

**What did you expect as result?**
Music playing :)

...

**Any other remarks**

...

**Please attach a Homey System log (Settings -> System -> Send Log)**

...

**Comment by bvdbos** [comment]
dupe of #1534

**Comment by satbopper** [comment]
this is a issue from 1.2.2,the homey mediaplayer together with sonos is not working anymore,idem in 1.3.0 rc7,this sofar for sonos playlists and if you have sonos speakers in pair

**Comment by psiebe** [comment]
does that mean I can bypass the issue temporarily by manually ungrouping my sonos speakers prior to asking homey to stream?

**Comment by bvdbos** [comment]
No, afaik that has got nothing to do with it... See bug #1534 ...

**Comment by satbopper** [comment]
thats possible ok but then it must be the sonos app,if i unpair because i have two play 1's i can play playlists with every speaker,soundcloud list number after number
sonos playlists stops after one number
further sonos,will not play numbers if i have paired the speakers
so some work for athom i guess

**Comment by jeroenvollenbrock** [comment]
@BasKiers message: 'HTTP response code 500 for "urn:schemas-upnp-org:service:AVTransport:1#RemoveAllTracksFromQueue"' }
2017-05-17 11:13:00 homey:manager:media            [Error: generic_playback_error]

**Comment by Povlov** [comment]
Having the issue "randomly"; 1.3.1, using soundcloud and youtube entries.

**Comment by bvdbos** [comment]
We seem to encounter this for mp3-streams too?

**Comment by Povlov** [comment]
I have given up on using the Music tab. 
There is not enough service available for me to employ this functionality.
Homey is a great home controller.

**Comment by bvdbos** [comment]
https://github.com/bvdbos/com.sasteren.radio
https://github.com/bvdbos/com.sasteren.podcast
:)

**Comment by bvdbos** [comment]
@Povlov These are issues with upnp, not with Homey Music in general...

**Comment by chrisboer** [comment]
Get an issue also but it is playing on a different speaker than I selected.

![afbeelding](https://user-images.githubusercontent.com/23163409/35071036-b38662e4-fbdf-11e7-8769-84a39fb6144e.png)

**Comment by michel-bos** [comment]
Hi all, there seems to be no progress on this issue. I can not use the Music Functionality either.

Cn anybody of Athom respond on this issue? What is the plan, if any?

Thanks for any response!

Michel
